app/controllers/rankings_controller.rb

```diff
@@ -22,6 +24,28 @@ def valid_secret_id?
     params[:secret_id] == @participant.secret_id
   end
 
+  def create_rankings_hash
+    @rankings_hash = Hash.new
+    @rankings.each do |ranking|
+      unless ranking.project_time.is_date
```

Comment:
nice! thanks for adding this check. also I like how you use `ranking.project_time`, good use of Rails convention!